manager/src/test/java/com/redhat/service/smartevents/manager/ProcessorServiceConnectorTest.java

```diff
@@ -98,11 +99,14 @@ void createConnectorSuccess() {
         ProcessorRequest processorRequest = new ProcessorRequest("ManagedConnectorProcessor", slackAction);
 
         //Emulate successful External Connector creation
-        Connector externalConnector = stubbedExternalConnector("connectorExternalId");
+        Connector externalConnector = spy(stubbedExternalConnector("connectorExternalId"));
```

Comment:
Why the spy here?

Reply:
Because I was mocking invocation of it's `getConnector()` on L#108 but this can be better achieved by subclassing it and overriding the method. I cannot _set_ the connector value when constructing the value as it does not exist until runtime and needs to be read from the database.